### ISSUE TITLE
StatsBase requirement

### ISSUE THREAD

**Issue by karanbudhraja-tgen** [milanflach/MultivariateAnomalies.jl#5]
It seems that StatsBase is still needed to be mentioned as a requirement (I noticed that it was removed in an earlier commit). I got the following error after adding the package as a requirement for a module (I'm using Pkg.instantiate).
```
ERROR: LoadError: ArgumentError: Package MultivariateAnomalies does not have StatsBase in its dependencies:
- If you have MultivariateAnomalies checked out for development and have
  added StatsBase as a dependency but haven't updated your primary
  environment's manifest file, try `Pkg.resolve()`.
- Otherwise you may need to report an issue with MultivariateAnomalies
Stacktrace:
 [1] require(::Module, ::Symbol) at ./loading.jl:836
 [2] include at ./boot.jl:326 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include(::Module, ::String) at ./sysimg.jl:29
 [5] top-level scope at none:2
 [6] eval at ./boot.jl:328 [inlined]
 [7] eval(::Expr) at ./client.jl:404
 [8] top-level scope at ./none:3
```

**Comment by meggart** [comment]
Which version of the package are you using? From looking at the [Project.toml](https://github.com/milanflach/MultivariateAnomalies.jl/blob/master/Project.toml) StatsBase seems to be included and I can not reproduce the error you are reporting.

**Comment by karanbudhraja-tgen** [comment]
In my Project.toml I have
```
MultivariateAnomalies = "6b1494d1-c0e4-5e11-970b-cb64accc6d0e"
```
and in my Manifest.toml I have
```
[[MultivariateAnomalies]]
deps = ["Combinatorics", "Distances", "LIBSVM", "LinearAlgebra", "Statistics", "Test"]
git-tree-sha1 = "66f810ae2359bc42b4a75ae4764a260a8effacb9"
uuid = "6b1494d1-c0e4-5e11-970b-cb64accc6d0e"
version = "0.2.3"
```
My understanding is that StatsBase should have appeared in `deps`. I am using Julia version `1.1.1`.

**Comment by meggart** [comment]
Ok this is strange. It look like in the registry the dependency does not match the Project.toml file, as seen here: https://github.com/JuliaRegistries/General/blob/master/M/MultivariateAnomalies/Deps.toml

So maybe we should just make a new release and tag a new version 0.2.4 to fix this. @milanflach did you already install JuliaRegistrator and TagBot?

**Comment by milanflach** [comment]
Yes, indeed. Thanks for mentioning this issue. If you checkout the latest version on master, it seems to work for me.
```Julia
 add MultivariateAnomalies#master
```
I will tag a new version to fix this issue.

**Comment by milanflach** [comment]
Solved with release [v0.2.4](https://github.com/milanflach/MultivariateAnomalies.jl/releases/tag/v0.2.4)
